### ISSUE TITLE
font face and size do not update automatically on new page load

### ISSUE THREAD

**Issue by AmenJlili** [basshelal/Wudooh#1]
Behavior: 
1. Set font face and size and observe page rendering as expected. 
2. Browse to a new page via any link. 
3. Font and size settings are retained by UI but new page's font face and font size are not updated.
Expected behavior: All pages during a browsing activity must automatically update font face and size.

![Test](https://user-images.githubusercontent.com/16106587/58758323-bbc6ad00-84ce-11e9-88a2-e6fb5c4bbdb0.gif)
.

**Comment by basshelal** [comment]
@jliliamen Yeah sorry about that, I noticed this too when installing it on my machine from the store.
Tweak ALL the settings (sliders, font and switches) and then reload the page, everything should work from there and behavior should be normal.

The issue is for some reason Chrome wasn't providing it the default values I gave it, I've updated this now but it won't be a few days until that version goes live.
Tell me if it works for you after this.
Sorry again, I'm completely new to Chrome extensions
